---
title: Der Throw-Operand muss von System.Exception abgeleitet werden.
ms.date: 07/20/2015
f1_keywords:
- vbc30665
- bc30665
helpviewer_keywords:
- BC30665
ms.assetid: 7c228087-39ea-4b30-a410-6ba711e67e5e
ms.openlocfilehash: 92fccbd2faac5580d28fef8cc02b51b23715f136
ms.sourcegitcommit: f8c270376ed905f6a8896ce0fe25b4f4b38ff498
ms.translationtype: MT
ms.contentlocale: de-DE
ms.lasthandoff: 06/04/2020
ms.locfileid: "84414945"
---
# <a name="throw-operand-must-derive-from-systemexception"></a>Der Throw-Operand muss von System.Exception abgeleitet werden.
Das für `Throw` angegebene Argument muss entweder eine Instanz von `System.Exception` oder eine Instanz einer von `System.Exception`abgeleiteten Klasse sein.  
  
 **Fehler-ID:** BC30665  
  
## <a name="to-correct-this-error"></a>So beheben Sie diesen Fehler  
  
- Verwenden Sie ein Argument, das von `System.Exception`abgeleitet wird, wie im folgenden Beispiel gezeigt.  
  
    ```vb
    Throw New System.Exception("This is an error.")  
    ```  
  
## <a name="see-also"></a>Weitere Informationen

- [Throw-Anweisung](../language-reference/statements/throw-statement.md)
- [Try...Catch...Finally-Anweisung](../language-reference/statements/try-catch-finally-statement.md)
- <xref:System.Exception?displayProperty=nameWithType>
- [Behandeln und Auslösen von Ausnahmen in .NET](../../standard/exceptions/index.md)
